src/components/pages/chefProfile/index.tsx

```diff
@@ -11,6 +12,7 @@ export default function ChefProfilePage(): ReactElement {
     return (
         <VStack className="w-full">
             <PEHeader />
+            <PEHeaderMobile />
```

Comment:
Here is the correct place where we check if we are on mobile or desktop, not within the components themselves:

```suggestion
            {isDesktop && <PEHeader />}
            {isMobile && <PEHeaderMobile />}
```

Reply:
Same thing for every other page